core/controlplane/config/templates/cloud-config-controller

```diff
@@ -3642,9 +3726,12 @@ write_files:
                   - /cluster-proportional-autoscaler
                   - --namespace=kube-system
                   - --configmap=kube-dns-autoscaler
+                  {{- if eq .KubeDns.Provider "coredns" }}
+                  - --target=Deployment/coredns
+                  {{- else }}
                   - --target=Deployment/kube-dns
+                  {{- end }}
                   - --default-params={"linear":{"coresPerReplica":{{ .KubeDns.Autoscaler.CoresPerReplica }},"nodesPerReplica":{{ .KubeDns.Autoscaler.NodesPerReplica }},"min":{{ .KubeDns.Autoscaler.Min}}}}
-                  - --logtostderr=true
```

Comment:
Was `logtostderr` intentionally removed here?

Reply:
No, at least for me! Good catch.